policyengine_us/parameters/gov/states/oh/tax/income/credits/lump_sum_distribution/rate.yaml

```diff
@@ -0,0 +1,359 @@
+description: Ohio multiplies this credit amount by this rate, based on age.
+metadata:
+  type: single_amount 
+  threshold_unit: year
+  period: year
+  amount_unit: /1
+  label: Ohio lump sum distribution credit rate
+  reference:
+    - title:  Ohio 2021 Instructions for Filing Original and Amended Individual Income Tax (IT 1040) and  School District Income Tax (SD 100)
+      href: https://tax.ohio.gov/static/forms/ohio_individual/individual/2021/pit-it1040-booklet.pdf#page=20
+    - title: Ohio Laws and Administrative Rules Section 5747.055 (G)
```

Comment:
need specific paragraph or subchapter index

Reply:
and I think the reference for the breakdown should include this: https://tax.ohio.gov/static/forms/ohio_individual/individual/2021/pit-it1040-booklet.pdf#page=29